### ISSUE TITLE
No person maintain this project?

### ISSUE THREAD

**Issue by lanzhiheng** [sparksuite/simplemde-markdown-editor#740]
I saw that the most recent commit is 3 years ago, is it this project abandoned?

**Comment by Ionaru** [comment]
There are a number of forks of this project still active and being maintained.

Among them are https://github.com/Ionaru/easy-markdown-editor and https://github.com/Inscryb/inscryb-markdown-editor, they are both very active.

**Comment by lanzhiheng** [comment]
@Ionaru Oh, Cool. Thx. I think because of the stars number. google is hard to search them.

**Comment by ruibaby** [comment]
@Ionaru wooow,Thx.

**Comment by lypwig** [comment]
Pro-tip: when you discover an unmaintained project, you can check it's [forks page](https://github.com/sparksuite/simplemde-markdown-editor/network/members) (by clicking on the numbers of forks), and look for the fork with the most of sub-forks, which is a good popularity indicator. In this case it's [EasyMDE](https://github.com/Ionaru/easy-markdown-editor).

**Comment by huioo** [comment]
444

**Comment by BelenCebrian** [comment]
> 
> 
> Pro-tip: when you discover an unmaintained project, you can check it's [forks page](https://github.com/sparksuite/simplemde-markdown-editor/network/members) (by clicking on the numbers of forks), and look for the fork with the most of sub-forks, which is a good popularity indicator. In this case it's [EasyMDE](https://github.com/Ionaru/easy-markdown-editor).

I agree with what @roipoussiere said, and I want to add that there's a Github project to make easier to compare forks: [active-forks](https://github.com/techgaun/active-forks/). It even has a online page in https://techgaun.github.io/active-forks/index.html when you can paste a repo's link and sort forks by stars, number of sub-forks, latest change, etc. Very useful.
